SA3/qa_web/static/css/qa.css

```diff
@@ -0,0 +1,168 @@
+    width: 100%;
+    height: 40px;
+	margin-top: 1px;
+}
+
+.page-header .btn{
+	font-family: 'Oxygen', cursive;
+}
+
+h1 {font-family: 'Nanum Brush Script', cursive;}
+
+reward {
+  font-size: 15px;
+  color: #FF4444;
+}
+
+.ans {
+  height: 40px;
+  width: 75px;
+  color:#500000;
+  text-align: center;
+  text-align: center;
+  vertical-align: middle;
+}
+
+tag {
+  background-color:#E0E0E0;
+  color:#500000;
+}
+
+
+#search {
+  position: fixed;
+  top: 0px;
+  left: 0px;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.7);
+
+  -webkit-transition: all 0.5s ease-in-out;
+  -moz-transition: all 0.5s ease-in-out;
+  -o-transition: all 0.5s ease-in-out;
+  -ms-transition: all 0.5s ease-in-out;
+  transition: all 0.5s ease-in-out;
+
+  -webkit-transform: translate(0px, -100%) scale(0, 0);
+  -moz-transform: translate(0px, -100%) scale(0, 0);
+  -o-transform: translate(0px, -100%) scale(0, 0);
+  -ms-transform: translate(0px, -100%) scale(0, 0);
+  transform: translate(0px, -100%) scale(0, 0);
+
+  opacity: 0;
+}
+
+#search.open {
+  -webkit-transform: translate(0px, 0px) scale(1, 1);
+  -moz-transform: translate(0px, 0px) scale(1, 1);
+  -o-transform: translate(0px, 0px) scale(1, 1);
+  -ms-transform: translate(0px, 0px) scale(1, 1);
+  transform: translate(0px, 0px) scale(1, 1);
+  opacity: 1;
+}
+
+#search input[type="search"] {
+  position: absolute;
+  top: 50%;
+  width: 100%;
+  color: rgb(255, 255, 255);
+  background: rgba(0, 0, 0, 0);
+  font-size: 60px;
+  font-weight: 300;
+  text-align: center;
+  border: 0px;
+  margin: 0px auto;
+  margin-top: -51px;
+  padding-left: 30px;
+  padding-right: 30px;
+  outline: none;
+}
+#search .btn {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin-top: 61px;
+  margin-left: -45px;
+}
+#search .close {
+  position: fixed;
+  top: 15px;
+  right: 15px;
+  color: #fff;
+  background-color: #428bca;
+  border-color: #357ebd;
+  opacity: 1;
+  padding: 10px 17px;
+  font-size: 27px;
+}
+
+orange {
+  color: #ff8a00;
+}
+
+blue {
+  color: #9fa6ff;
+}
+
+a:hover {color: #FF8800; }
+
+.closed {
+  width:100%;
+  height:7px;
+  position: absolute;
+
+  background: #FF4444;
+}
+
+.open {
+  width:100%;
+  height:7px;
+  position: absolute;
+
+  background: #99CC00;
+}
+
+.rewarding {
+  width:100%;
+  height:7px;
+  position: absolute;
+
+  background: #AA66CC;
+}
+.custom-big {
+  font-size:20px;
+  padding : 2px 0px 2px 0px;
+}
+cool {
+  font-family: 'Lobster', cursive;
+}
+.custom-code {
+    font-family: Monaco, Consolas, "Andale Mono", "DejaVu Sans Mono", monospace;
+    font-size: 95%;
+    line-height: 140%;
+    white-space: pre;
+    white-space: pre-wrap;
+    white-space: -moz-pre-wrap;
+    white-space: -o-pre-wrap;
+    background: #faf8f0;
+}
+
+#content code {
+    display: block;
+    padding: 0.5em 1em;
+    border: 1px solid #bebab0;
+}
+.topbar {
+  width:100%;
+  height:7px;
+  position: absolute;
+  z-index:1000;
+
+  background: #9fa6ff;
+}
```

Comment:
<img width="134" alt="screen shot 2018-02-16 at 11 09 54 am" src="https://user-images.githubusercontent.com/31190776/36317022-ee9ffd5e-1309-11e8-8ea4-b87e1cc481bd.png">

Reply:
Not sure what is meant by line 168, as it is viewed as an odd icon.

Reply:
@13rinda It seems it is a good style to always put the newline as a last character. I added one. Thanks.